app/src/main/java/com/teamwss/websoso/ui/activityDetail/ActivityDetailActivity.kt

```diff
@@ -52,16 +53,37 @@ class ActivityDetailActivity :
     }
     private val userId: Long by lazy { intent.getLongExtra(USER_ID_KEY, DEFAULT_USER_ID) }
     private var _popupWindow: PopupWindow? = null
+    private lateinit var activityResultCallback: ActivityResultLauncher<Intent>
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setupActivityResultCallback()
         setupUserIDAndSource()
         setActivityTitle()
         setupMyActivitiesDetailAdapter()
         setupObserver()
         onBackButtonClick()
     }
 
+    private fun setupActivityResultCallback() {
+        activityResultCallback = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            when (result.resultCode) {
+                ResultFrom.FeedDetailBack.RESULT_OK, ResultFrom.CreateFeed.RESULT_OK -> {
+                    activityDetailViewModel.updateRefreshedActivities()
+                }
+                ResultFrom.FeedDetailRemoved.RESULT_OK -> {
+                    activityDetailViewModel.updateRefreshedActivities()
+                    showSnackBar(getString(R.string.feed_removed_feed_snackbar))
+                }
+                ResultFrom.BlockUser.RESULT_OK -> {
+                    activityDetailViewModel.updateRefreshedActivities()
+                    val nickname = result.data?.getStringExtra(USER_NICKNAME).orEmpty()
+                    showSnackBar(getString(R.string.block_user_success_message, nickname))
+                }
+            }
+        }
+    }
```

Comment:
r: 다른 화면에 구현한 것 처럼 이렇게 정의해두는건 어떤가요

```suggestion
    private val activityResultCallback: ActivityResultLauncher<Intent> = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
            when (result.resultCode) {
                ResultFrom.FeedDetailBack.RESULT_OK, ResultFrom.CreateFeed.RESULT_OK -> {
                    activityDetailViewModel.updateRefreshedActivities()
                }
                ResultFrom.FeedDetailRemoved.RESULT_OK -> {
                    activityDetailViewModel.updateRefreshedActivities()
                    showSnackBar(getString(R.string.feed_removed_feed_snackbar))
                }
                ResultFrom.BlockUser.RESULT_OK -> {
                    activityDetailViewModel.updateRefreshedActivities()
                    val nickname = result.data?.getStringExtra(USER_NICKNAME).orEmpty()
                    showSnackBar(getString(R.string.block_user_success_message, nickname))
                }
            }
        }
    }
```